ui/src/app/components/workflows/workflow/dynamic-parts/string-sequence-part/string-sequence-part.component.html

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<clr-control-container *ngFor="let val of value; let i = index; trackBy: trackByFn;">
+  <label *ngIf="i == 0">{{name}}</label>
+  <input clrControl [readOnly]="isShow" type="text" [size]="70" [ngModel]="val" (ngModelChange)="ngModelChanged($event, i)"/>
```

Comment:
The size should be customizable. Some of the app arguments are cut off

Reply:
Yes, I agree. Let's do it in the next sprint.